packages/html/test.txt

```diff
@@ -0,0 +1,103 @@
+"Backspace", "Backspace"
```

Comment:
Do we need this file?

Reply:
No, it was accidentally committed, I'll remove it.